lib/hls/hls_parser.js

```diff
@@ -1640,7 +1640,7 @@ shaka.hls.HlsParser = class {
           segmentResponse.data, initSegmentResponse.data);
     }
 
-    if (mimeType == 'video/mp2t') {
+    if (mimeType && mimeType.toLowerCase() == 'video/mp2t') {
```

Comment:
There are other `==` checks above that would need to get fixed too.  How about making it lowercase when it is read from the HLS file?

Reply:
@TheModMaker I am not sure could this be applicable for all mime type standards. Maybe there are standards with uppercase so this proposal couldn’t fit.